JoinColony/colonyDapp
# Add locked values

## Description

It is not possible to edit values of an Expenditure, while it is in a locked state.

<img width="1343" alt="Screenshot 2022-06-30 at 07 39 20" src="https://user-images.githubusercontent.com/91876137/176601043-c8dfd965-77d5-4a1f-853e-59066a91ddc9.png">

[Figma design:](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=2420%3A1087531)

## TESTING

1. Ensure you have values in every field.
2. Click the 'Submit draft' button.
3. In the modal click the 'Stake' button.
4. Click the 'Lock values' button.
5. You should then be in the locked state.

## TODO

- add button, which revert the expenditure to an editable state.

Resolves to #3441
